src/TextInput/TextInput.tsx

```diff
@@ -18,8 +18,11 @@ interface Props extends CommonFieldProps {
   onBlur?: (e: FocusEvent<HTMLInputElement>) => void
 }
 
+type AutoComplete = 'off' | 'bday-day	' | 'bday-month' | 'bday-year'
```

Comment:
```suggestion
type AutoComplete = 'off' | 'bday-day' | 'bday-month' | 'bday-year'
```
is this whitespace needed?